snuba/web/views.py

```diff
@@ -108,12 +108,20 @@ def check_down_file_exists() -> bool:
             return False
 
 
-def check_clickhouse() -> bool:
+def check_clickhouse(filter_experimental: bool = True) -> bool:
```

Comment:
Very much a nit but `ignore_experimental` seems more intuitive to me

Reply:
you're right it is